### ISSUE TITLE
do we need element.elementBy(condition) as shortcut for element.all('*').elementBy(condition)

### ISSUE THREAD

**Issue by yashaka** [KnowledgeExpert/selenidejs#83]
Seems like good idea to have such helper-alias... But why not be just KISS and use more verbose version?

**Comment by AleksanderPopov** [comment]
makes sense to me, should be not so hard on understand since we same function in collection...

agree on building that alias?

@yashaka
